packages/assets-controllers/CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `TokenListController` now exports a `TokenListControllerMessenger` type ([#3609](https://github.com/MetaMask/core/pull/3609)).
+- `TokenDetectionController` exports types `TokenDetectionControllerMessenger`, `TokenDetectionControllerActions`, `TokenDetectionControllerGetStateAction`, `TokenDetectionControllerEvents`, `TokenDetectionControllerStateChangeEvent` ([#3609](https://github.com/MetaMask/core/pull/3609)).
+
+### Changed
+- **BREAKING:** `TokenDetectionController` is upgraded to extend `BaseControllerV2` and `StaticIntervalPollingController` ([#3609](https://github.com/MetaMask/core/pull/3609)).
+  - The constructor now expects an options object as its only argument, with required properties `messenger`, `chainId`, required callbacks `onPreferencesStateChange`, `getBalancesInSingleCall`, `addDetectedTokens`, `getTokensState`, `getPreferencesState`, and optional properties `interval`, `selectedAddress`.
+  - The controller state is an empty object and the controller config object is deprecated.
```

Comment:
"The controller state is an empty object" — this is slightly confusing. Wasn't state empty by default before as well? Or do you mean something else?

Reply:
The controller state previously had the `disabled` property, which has been moved to a private class field. But I'm good with removing this bullet point if it isn't providing useful info, and we can expect readers to have some understanding of how v1 and v2 controllers differ.

Reply:
Okay, perhaps we want to merge this bullet point with the first bullet point? Something like "This options object should be used instead of `config`".

Reply:
Fixed here: https://github.com/MetaMask/core/pull/3609/commits/b2a5af54274d3314318c8d2f5cc4481e6b0454e6